app/src/main/java/com/example/trempelapp/presentation_layer/splash_screen/TrempelSplashActivity.kt

```diff
@@ -42,11 +42,11 @@ class TrempelSplashActivity : BaseActivity() {
         trempelSplashViewModel.isLoggedInLiveData.observe(this, {
 
             if (trempelSplashViewModel.isLoggedInLiveData.value == false) {
-                val intent = Intent(this, LogInActivity::class.java)
+                val intent = Intent(this, TrempelLogInActivity::class.java)
                 startActivity(intent)
```

Comment:
boilerplate code.
You are using the same code on lines 46 and 50, 47 and 51

Reply:
done